NuGet.config

```diff
@@ -3,6 +3,10 @@
   <packageSources>
     <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
     <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
+    <!-- Workaround for missing package on our feed -->
+    <add key="DotNetCore" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
+    <!-- Additional feeds for dotnet cache -->
+    <add key="Roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
```

Comment:
Won't need this yet, since it's pulled in via M.N.A for apps but we will need it once we move to 2.0.